test/FsAutoComplete.IntegrationTests/DotNetCore/sample1/l1/l1.fsproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <MyCustomProp>custom value</MyCustomProp>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="Library.fs" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="FSharp.NET.Sdk" Version="1.0.*" PrivateAssets="All" />
+    <PackageReference Include="FSharp.Core" Version="4.1.*" />
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+  </ItemGroup>
+  <Import Project="..\..\..\..\..\.paket\Paket.Restore.targets" />
```

Comment:
can you remove Paket target for now? so we can test a normal dotnet new project.
we can add paket later. just to remove a possibile failure